src/modal/sign-txn/_pera-wallet-sign-txn-modal.scss

```diff
@@ -0,0 +1,17 @@
+.pera-wallet-sign-txn-modal.pera-wallet-modal {
+  .pera-wallet-modal__body {
+    width: 480px;
+    height: 640px;
+
+    #pera-wallet-iframe {
+      position: absolute;
```

Comment:
Let's override the unwanted modal styles instead of absolute positioning

Reply:
Now I remember exactly like we have to use absolute here to override the content here. We can hide the desktop-mode here, maybe with css, but since the iframe is inside it, if we use `display:none;`, iframe will disappear too. It seems like absolute is required in the create account and select-account section on the connect flow side. But I removed absolute from sign txn modal.
<img width="558" alt="Screen Shot 2022-09-08 at 15 14 49" src="https://user-images.githubusercontent.com/54077855/189120879-0705d030-9213-4761-b281-553bcc551be3.png">